.clang-format

```diff
@@ -0,0 +1,72 @@
+---
+Language:        Cpp
+# BasedOnStyle:  LLVM clang-format 4.0
+AccessModifierOffset: -4
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlinesLeft: false
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: Inline
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: false
+BinPackParameters: false
+BreakBeforeBinaryOperators: None
+BreakBeforeBraces: Stroustrup # Attach
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializersBeforeComma: false
+ColumnLimit:     80
+CommentPragmas:  '^ IWYU pragma:'
+ConstructorInitializerAllOnOneLineOrOnePerLine: false
+ConstructorInitializerIndentWidth: 4
+ContinuationIndentWidth: 8
```

Comment:
Actually, I checked and continuations are being handled by AlignAfterOpenBracket, so it's unfortunately coupled between whether blocks are aligned (AlignAfterOpenBracket: DontAlign) and whether the first parameter is aligned with the other parameters (AlignAfterOpenBracket: AlwaysBreak). It seems like whoever made that a ternary (circa 3.5) didn't notice this, or perhaps wanted it this way.